### PR TITLE
[RHELC-1289] Add a one key conversion method

### PR DESCRIPTION
**Pull request by danmyway**
* add a full conversion test validating, that systems pre-registered to the satellite with one activation key containing both original OS and RHEL repos work

Jira Issues:

- [RHELC-1289](https://issues.redhat.com/browse/RHELC-1289)
- 

Checklist

- [x] PR has been tested manually in a VM (either author or reviewer)
- [x] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [x] GitHub label has been added to help with Release notes 
- [x] PR title explains the change from the user's point of view
- [x] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [x] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

**Review by kokesak — APPROVED** [review]

File: tests/integration/tier0/destructive/conversion-method/test_one_key_satellite.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    Conversion method using the Satellite credentials for a registration.
+    The system is pre-registered to the Satellite instance prior to the conversion.
+    We use one activation key containing both the original OS and RHEL repositories.
+    """
+    # We need to enable the current system repositories prior to the conversion
+    shell("subscription-manager repos --enable=*")
+
+    with convert2rhel("-y --debug") as c2r:
+        c2r.expect("Conversion successful!")
+    assert c2r.exitstatus == 0
```
[the remark this answers is not in the document]
Comment (r0x0d) [reply tests/integration/tier0/destructive/conversion-method/test_one_key_satellite.py]: more docstrings than lines of test

**Pull request merged**